src/general/storageclass/constraint.tmpl

```diff
@@ -0,0 +1,25 @@
+apiVersion: templates.gatekeeper.sh/v1
+kind: ConstraintTemplate
+metadata:
+  name: k8sstorageclass
+  annotations:
+    description: >-
+      Requires storage classes to be specified when used.
```

Comment:
can we add min version of GK required here too?

Reply:
As a comment, or is there a different way to do that?

Reply:
Oh. In the description? kk.